test/fixtures/cookbooks/test/recipes/openjdk.rb

```diff
@@ -0,0 +1,3 @@
+openjdk_install node['version'] do
+  url "http://192.168.0.35:8080/openjdk-#{node['version']}_linux-x64_bin.tar.gz"
```

Comment:
This is temporary. Saves me having to redownload them from the net every 5 minutes